docs/topics/mta-web-proxy-config.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-proxy-config_{context}"]
+= Configuring HTTP and HTTPS proxy settings
+
+This management module configures HTTP & HTTPS proxy settings.
+
+.Procedure
+
+. Click the radio button to enter the following information.
+. In the left menu of the *Administrator* view, click *Proxy*.
+. Click *HTTP proxy* or *HTTPS proxy* toggle switches to enable the proxy connection.
```

Comment:
Use "Toggle" as the verb. Either "Toggle the *x* switch or the *y* switch.." or "Toggle *x* or *y* to..." I'd got woth the 2nd, but your call.